emscripten-releases-tags.txt

```diff
@@ -1,10 +1,11 @@
 {
-  "latest": "2.0.20",
+  "latest": "2.0.21",
   "releases": {
-    "2.0.20-lto": "d1b26cd17e51c5c705eea69b9545975e3705c058",
+    "2.0.21": "72f4ec97fbc7ec16c15ae68a75b0a257b2835160",
     "2.0.20": "e0c15cd14170f407a9eb27fcbad22931dc67feb7",
-    "2.0.19-lto": "4487f6c5107e7882ae2bad6d26c34ffdceb713f0",
+    "2.0.20-lto": "d1b26cd17e51c5c705eea69b9545975e3705c058",
     "2.0.19": "9b9ff2dabfb4a7fbacbc004c0bead12a60f9d05c",
+    "2.0.19-lto": "4487f6c5107e7882ae2bad6d26c34ffdceb713f0",
```

Comment:
Do you know why the order changed here?

Reply:
Maybe we added the LTO entries manually, and when the script regenerated the file, it re-sorted?